crates/sui-sdk/src/apis.rs

```diff
@@ -456,6 +873,9 @@ impl EventApi {
             .await?)
     }
 
+    /// Return a stream of events based on the given event filter.
+    ///
+    /// By default, each page will return 100 items.
```

Comment:
Not sure I understand the comment about pages being 100 items long -- this function returns a stream, so the consumer has no idea that it will make requests in batches of 100 (it also can't control that -- it's an implementation detail).

Reply:
You're right, don't know why I misunderstood this.